data/helper/data_dictionary.csv

```diff
@@ -35,9 +35,9 @@ Text,publication_date,date of first publication in original language,,2017-07-04
 Text,publishing_house,name of publishing house,provide pinyin for Chinese publishing houses,2017-07-04,LH,2017-07-04,LH
 Text,first_chin_edition,year of first edition in China,,2017-07-04,LH,2017-07-04,LH
 Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,,,,
-Text,neibu,yes/no whether this was originally a neibu publication,,2017-07-04,LH,2017-07-04,LH
+Text,neibu,"yes/no whether this was originally a neibu publication, add „0“ for every publication starting from 1980",,2017-07-04,LH,2017-07-04,LH
```

Comment:
avoid the use of U+201E/F  ``„‟`` in english, it messes up the auto-parsing of tables, use two U+0027 (singel-quote) ``''`` inside text cells, or U+0022 ``""`` if you absolutely have to use multiple nested quotation marks.